spyderlib/plugins/shortcuts.py

```diff
@@ -807,6 +816,22 @@ def apply_settings(self, options):
         self.table.save_shortcuts()
         self.main.apply_shortcuts()
 
+    def load_button_clicked(self, change):
+        selected = self.combo_keybindings.currentText().lower()
+        reset_shortcuts()

```

Comment:
I think we should preserve customizations for any scheme, and leave the button called `Reset to default schemes` :-)